console/src/components/Common/CustomInputTypes/JsonInput.js

```diff
@@ -81,28 +79,26 @@ const JsonInput = props => {
 
   const getJsonEditor = () => {
     return (
-      <AceEditor
-        key="ace_json_editor"
-        {...allProps}
-        mode="json"
-        theme="github"
-        name="jsontoggler"
-        minLines={10}
-        maxLines={100}
-        width="100%"
-        value={data}
-        showPrintMargin={false}
-        onChange={handleTextAreaChangeAndPropagate}
-        showGutter={false}
-        focus
-        commands={[
-          {
-            name: 'toggleEditor',
-            bindKey: { win: 'Ctrl-Space', mac: 'Command-Space' },
-            exec: handleEditorExec,
-          },
-        ]}
-      />
+        <ResizableAceEditor
+          id="ace_json_editor"
+          allProps={allProps}
+          mode="json"
+          theme="github"
+          name="jsontoggler"
+          minLines={10}
+          maxLines={100}
+          value={data}
+          showPrintMargin={false}
+          onChange={handleTextAreaChangeAndPropagate}
+          showGutter={false}
+          commands={[
+            {
+              name: 'toggleEditor',
+              bindKey: { win: 'Ctrl-Space', mac: 'Command-Space' },
+              exec: handleEditorExec,
+            },
+          ]}
```

Comment:
Since this `ResizableAceEditor` is mainly being used as a JSON editor my suggestion is that you could make a new component `JSONEditor` with all the _non-essential props_ filled in and while using it, you could only provide the `value` and the `onChange` method alone.